### DISCUSSION TITLE
Technical issue with formr email system

### DISCUSSION THREAD

**Discussion by shadowanger** [rubenarslan/formr.org#540] [General]
Dear Ruben,

I hope this message finds you well. I wanted to bring to your attention that we are currently experiencing an issue with the email function on formr. The status for all emails across multiple studies is showing as "email_queued" and seems to be stuck. Could you please look into this and help us resolve the issue at your earliest convenience?

Thank you for your attention to this matter.

Best regards,

Robin

**Comment by rubenarslan** [comment]
Working again? It does for me, not sure what got stuck.

**Reply by shadowanger** [reply]
It does work now. Thank you :)
